Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -83,18 +83,17 @@ extension BrowserViewController: RewardsUIDelegate {
 }
 
 extension BrowserViewController: RewardsDataSource {
-    func displayString(for url: URL) -> String? {
-        return url.host
-    }
-    
-    func retrieveFavicon(with url: URL, completion: @escaping (FaviconData?) -> Void) {
+    func retrieveFavicon(for pageURL: URL, faviconURL: URL?, completion: @escaping (FaviconData?) -> Void) {
         let favicon = UIImageView()
         DispatchQueue.main.async {
-            favicon.setIconMO(nil, forURL: url, completed: { color, url in
+            favicon.setIconMO(nil, forURL: pageURL, completed: { color, url in
```

Comment:
For sake of my current local changes can we change this to: `faviconURL ?? pageURL` instead of just `pageURL` (will just be blown away later anyways)

Reply:
yes